js/selectors/dashboard-selectors.js

```diff
@@ -5,6 +5,10 @@ import { SORT_BY_NAME, SORT_BY_MOST_PROGRESS, SORT_BY_LEAST_PROGRESS,
 import { compareStudentsByName } from "../util/misc";
 import { fromJS } from "immutable";
 
+const kSortGroupFirst = 1;
+const kSortGroupSecond = 2;
+const kSortGroupThird = 3;
```

Comment:
May be more helpful if the names were more descriptive like kSortGroupAwaitingFeedback, kSortGroupFeedbackGiven, and kSortGroupNotStarted.

Reply:
I was hoping the opposite - that these could be reused for any type of sort and all that we need to do is put some data into a group.  This way when comparing we know first always comes before second, second before third, etc.  But if I have names like `kSortGroupAwaitingFeedback` it doesn't really imply any sort meaning and it's up to someone reading the code to make sense of where it gets positioned.  Basically I'm trying to reduce cognitive load and make this sorting as easy as possible for engineers.